### PR TITLE
[SHELL32] pl-PL.rc: Fix IDD_FILE_PROPERTIES & IDD_FOLDER_PROPERTIES

### PR DESCRIPTION
**Pull request by JoachimHenze**
- This removes an ancient polish-only workaround/extrawurst of 0.4.13-dev-1171-g c81af08f51e0834c53573d6be90967373a3c25f0 which made the dialog bigger than allowed in y, and which reduced its usability on small monitor resolution.
- Sync the dialogs dimensions back to en-US and all other localizations. Notice also that both dialogs have the same height again afterwards.
- Maximizes the available space for the dynamic parts of the dialog on the right side. Historically even 160 turned out to be too short for that, e.g. EDITTEXT 14009, the dynamic part of "Lokalizacja:". and also EDITTEXT 14007, the dynamic part of "Program:". Therefore we used 170 everywhere for those widths in all other languages. But due to the polish workaround it was crippled to 155 here. Not good!
- Notice also how now the leftmost checkboxes position of the Attributes is perfectly aligned with the other dynamic content of that column above **after the fix**. This will be important in the future when we will restore the vertical layout of Attribute-checkboxes in both dlgs (Plik vs. checkbox, and Katalog vs. checkbox) to match 2k3sp2

JIRA issue: none

The dialog 8 **before** the fix:
old_dlg8

The dialog 8 **after** the fix:
new_dlg8

The dialog 9 **before** the fix:
old_dlg9

The dialog 9 **after** the fix:
new_dlg9

the_after_pic_showing_both_dlgs_at_runtime: Looks fine!
the_after_pic_showing_both_dlgs_at_runtime

### CODE REVIEW

**Comment by JoachimHenze** [comment]
Let me know if you know any native polish speaker in github for review.

**Comment by binarymaster** [comment]
Cc @reactos/lang-polish

File: dll/win32/shell32/lang/pl-PL.rc [hunk 1]
```diff
@@ -251,34 +251,34 @@ BEGIN
...
     EDITTEXT 14001, 58, 9, 170, 14, ES_LEFT | WS_TABSTOP
     LTEXT "", -1, 8, 29, 221, 1, SS_ETCHEDHORZ | WS_CHILD | WS_VISIBLE
-    LTEXT "Typ pliku:", 14004, 8, 35, 50, 10
+    LTEXT "Typ:", 14004, 8, 35, 50, 10
     CONTROL "Plik", 14005, "edit", ES_LEFT | ES_READONLY | NOT WS_BORDER | NOT WS_TABSTOP, 58, 35, 170, 10
-    LTEXT "Otwierany za pomocą:", 14006, 8, 48, 40, 16 //FIXME: That string doesn't fit. And enlarging the control would be foolish, as there is dynamic content on the right side of it. Just shrink the text, e.g. to just "Programy".
-    ICON "", 14025, 88, 52, 11, 10, NOT WS_VISIBLE
-    PUSHBUTTON "&Zmień...", 14024, 178, 50, 50, 14, WS_CHILD | WS_VISIBLE | WS_TABSTOP
-    EDITTEXT 14007, 58, 48, 115, 16, ES_LEFT | ES_READONLY | NOT WS_BORDER | ES_AUTOHSCROLL | NOT WS_TABSTOP
+    LTEXT "Programy:", 14006, 8, 53, 50, 10
```
Comment (archeYR) [inline dll/win32/shell32/lang/pl-PL.rc:264]:
```suggestion
    LTEXT "Program:", 14006, 8, 53, 50, 10
```
Singular form.

File: dll/win32/shell32/lang/pl-PL.rc [hunk 2]
```diff
@@ -251,34 +251,34 @@ BEGIN
...
+    EDITTEXT 14011, 58, 91, 170, 10, ES_LEFT | ES_READONLY | NOT WS_BORDER | NOT WS_TABSTOP
+    LTEXT "Na dysku:", 140112, 8, 107, 45, 10
+    EDITTEXT 14012, 58, 107, 170, 10, ES_LEFT | ES_READONLY | NOT WS_BORDER | NOT WS_TABSTOP
     LTEXT "", -1, 8, 123, 221, 1, SS_ETCHEDHORZ | WS_CHILD | WS_VISIBLE
     LTEXT "Utworzony:", 14014, 8, 131, 45, 10
-    EDITTEXT 14015, 73, 131, 155, 10, ES_LEFT | ES_READONLY | NOT WS_BORDER | NOT WS_TABSTOP
-    LTEXT "Zmodyfikowany:", 14016, 8, 147, 60, 10
-    EDITTEXT 14017, 73, 147, 155, 10, ES_LEFT | ES_READONLY | NOT WS_BORDER | NOT WS_TABSTOP
+    EDITTEXT 14015, 58, 131, 170, 10, ES_LEFT | ES_READONLY | NOT WS_BORDER | NOT WS_TABSTOP
+    LTEXT "Zmieniono:", 14016, 8, 147, 45, 10
```
Comment (archeYR) [inline dll/win32/shell32/lang/pl-PL.rc:278]:
```suggestion
    LTEXT "Zmieniony:", 14016, 8, 147, 45, 10
```
For consistency with other fields.

**Review by archeYR — CHANGES_REQUESTED** [review]

File: dll/win32/shell32/lang/pl-PL.rc (hunk 2 above)
Comment (JoachimHenze) [reply dll/win32/shell32/lang/pl-PL.rc:278]: Agreed, and applied. Resolved.

File: dll/win32/shell32/lang/pl-PL.rc (hunk 1 above)
Comment (JoachimHenze) [reply dll/win32/shell32/lang/pl-PL.rc:264]: Agreed, and applied.

File: dll/win32/shell32/lang/pl-PL.rc [hunk 3]
```diff
@@ -251,34 +251,34 @@ BEGIN
...
+IDD_FILE_PROPERTIES DIALOGEX 0, 0, 240, 205
 STYLE DS_SHELLFONT | WS_CHILD | WS_CAPTION
 CAPTION "Ogólne"
 FONT 8, "MS Shell Dlg", 0, 0, 0x0
 BEGIN
     ICON "", 14000, 16, 7, 32, 32, WS_VISIBLE
     EDITTEXT 14001, 58, 9, 170, 14, ES_LEFT | WS_TABSTOP
     LTEXT "", -1, 8, 29, 221, 1, SS_ETCHEDHORZ | WS_CHILD | WS_VISIBLE
-    LTEXT "Typ pliku:", 14004, 8, 35, 50, 10
+    LTEXT "Typ:", 14004, 8, 35, 50, 10
```
Comment (JoachimHenze) [inline dll/win32/shell32/lang/pl-PL.rc:262]:
```suggestion
    LTEXT "Typ:", 14004, 8, 35, 42, 10
```
width 42 (the answer to all questions) will work here as well, as proven by the other dlg.

**Comment by Matriks404** [comment]
I don't get these changes. I understand somewhat why would you want to shrink the labels to make window smaller, but Windows doesn't do that as far as I am aware. And I especially don't understand why is "Otwierany za pomocą" changed to "Program:". That makes no sense. Wouldn't it be better to translate it to "Otwórz w:" (perfective) or "Otwieraj w:" (imperfective and probably better)?

**Comment by archeYR** [comment]
> I don't get these changes. I understand somewhat why would you want to shrink the labels to make window smaller, but Windows doesn't do that as far as I am aware. And I especially don't understand why is "Otwierany za pomocą" changed to "Program:". That makes no sense. Wouldn't it be better to translate it to "Otwórz w:" (perfective) or "Otwieraj w:" (imperfective and probably better)?

Windows does adjust the window size depending on language, while ReactOS tends to keep the same window size for all. I suppose this is to reduce the burden when modifying the interface, as still many windows and dialogs are subject to change. While "Program:" seems obvious enough for me, "Otwieraj w" is a valid suggestion indeed. You may propose it via PR.

**Pull request merged**